randomColor/color.css

```diff
@@ -0,0 +1,43 @@
+.navBar {
```

Comment:
recommand  to use kebab-case style selector, such as .nav-bar, It is helpful to distinguish  from function names。